script/QuestFactory.s.sol

```diff
@@ -11,21 +12,16 @@ import {ProxyAdmin, ITransparentUpgradeableProxy} from "openzeppelin-contracts/p
 // # To Upgrade QuestFactory.sol run this command below
 // ! important: make sure storage layouts are compatible first:
 // bun clean && forge clean && forge build && npx @openzeppelin/upgrades-core validate --contract QuestFactory
-// forge script script/QuestFactory.s.sol:QuestFactoryUpgrade --rpc-url sepolia --broadcast --verify -vvvv
-contract QuestFactoryUpgrade is Script {
-    function run() external {
-        uint256 deployerPrivateKey = vm.envUint("MAINNET_PRIVATE_KEY");
+// bun sphinx propose script/QuestFactory.s.sol --target-contract QuestFactoryUpgrade --networks testnets
+contract QuestFactoryUpgrade is Script, SphinxConfig {
+    function run() external sphinx {
         ITransparentUpgradeableProxy questfactoryProxy = ITransparentUpgradeableProxy(C.QUEST_FACTORY_ADDRESS);
 
-        vm.startBroadcast(deployerPrivateKey);
-
-        ProxyAdmin(C.PROXY_ADMIN_ADDRESS).upgrade(questfactoryProxy, address(new QuestFactory()));
-
-        vm.stopBroadcast();
+        questfactoryProxy.upgradeTo(address(new QuestFactory{ salt: bytes32(0) }()));
```

Comment:
We deploy the `QuestFactory` via `CREATE2` because Sphinx doesn't support the `CREATE` opcode yet. (We're adding it soon)